.github/workflows/tests-evault-core.yml

```
@@ -0,0 +1,34 @@
+  pull_request:
+    branches: [main]
+    paths:
+      - 'infrastructure/w3id/**'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)